### PR TITLE
choose image format using defines

### PR DESCRIPTION
**Pull request by you-win**

### CODE REVIEW

**Review by j20001970 — CHANGES_REQUESTED** [review]

File: GDMP/io/camera_helper_cv.cc [hunk 1]
```diff
@@ -46,6 +46,11 @@ class CameraHelper::CameraHelperImpl : public cv::VideoCapture {
 			thread = std::thread([this, size]() -> void {
 #if !MEDIAPIPE_DISABLE_GPU
 				Ref<GPUHelper> gpu_helper = GPUHelper::_new(graph->get_gpu_resources().get());
+				auto cv_format = cv::COLOR_BGR2RGBA : cv::COLOR_BGR2RGB;
```
[the remark this answers is not in the document]
Comment (you-win) [reply GDMP/io/camera_helper_cv.cc:49]: Yep it is, I'm running into weird build errors locally with/without my changes, so I'm fixing that first and then will update this pr

[the remark this answers is not in the document]
Comment (j20001970) [reply GDMP/io/camera_helper_cv.cc:49]: Hi, is there any update on the pr?

[the remark this answers is not in the document]
Comment (you-win) [reply GDMP/io/camera_helper_cv.cc:49]: Sorry, I've been busy recently. I will try and get around to fixing this when I can.

**Comment by j20001970** [comment]
> This was working before on `master` but now when I'm compiling, I get this error
> 
> ```
> ERROR: C:/users/theaz/_bazel_theaz/7hqmoffz/external/com_github_gflags_gflags/BUILD:16:30: Executing genrule @com_github_gflags_gflags//:gflags_google_h failed: (Exit -1): bash.exe failed: error executing command C:\Users\theaz\scoop\apps\msys2\current\usr\bin\bash.exe -c ... (remaining 1 argument skipped)
> Action failed to execute: java.io.IOException: ERROR: src/main/native/windows/process.cc(202): CreateProcessW("C:\Users\theaz\scoop\apps\msys2\current\usr\bin\bash.exe" -c "source external/bazel_tools/tools/genrule/genrule-setup.sh; awk '{ gsub(/@ns@/, \"google\"); gsub(/@NS@/, \"GOOGLE\"); print; }' external/com_github_gflags_gflags/src/gflags_ns.h.in > bazel-out/x64_windows-opt/bin/external/com_github_gflags_gflags/gflags_google.h"): The system cannot find the file specified.
>  (error: 2)
> ```
> 
> The referenced directory is empty. I have tried deleting the bazel cache but there error persists. I'm also not sure where to apply a fix. The `WORKSPACE` file refers to a valid zip archive when downloading gflags, so there must be an intermediate step that is failing to generate the `gflags_google.h` file.

That's strange, I'm still able to compile on master branch with my Windows VM. I will clear bazel workspace and try again to see if I can reproduce the issue.

**Comment by j20001970** [comment]
Update: I'm able to compile the code by updating mediapipe repo to v0.8.10.2, after encountering a issue about missing symbols from `bazel_tools`.
I have updated `mediapipe` submodule for GDMP, please pull latest commit, then clear mediapipe workspace with
`bazelisk clean --expunge` and try again, that bazel version should be 5.2.0

**Comment by j20001970** [comment]
> Update: I'm able to compile the code by updating mediapipe repo to v0.8.10.2, after encountering a issue about missing symbols from `bazel_tools`. I have updated `mediapipe` submodule for GDMP, please pull latest commit, then clear mediapipe workspace with `bazelisk clean --expunge` and try again, that bazel version should be 5.2.0

Does the error persist on latest commit?

**Comment by you-win** [comment]
@j20001970 There was an additional error on my end where `bazel` was trying to use an `msys2` installation at an invalid path. I am able to compile now.

Will find my test project and test the results.

**Comment by you-win** [comment]
Sorry for the delay, I have tested this with OpenCV 3.4.10 using `//mediapipe/graphs/pose_tracking:pose_tracking_cpu_deps` and it seems to work.

**Review by j20001970 — APPROVED** [review]

**Comment by j20001970** [comment]
Thanks!

**Pull request merged**